docs/examples/userguide/language_basics/parameter_refcount.pyx

```diff
@@ -0,0 +1,20 @@
+from __future__ import print_function
+
+from cpython.ref cimport PyObject
+
+import sys
+
+python_string = "foo"
+python_string_refcount = sys.getrefcount(python_string)
```

Comment:
This is a bit risky in a test. Python string literals are interned by Cython, which means that their exact refcount might depend on other modules, and therefore not be predictable. You can avoid that by constructing a new string object with some non-optimisable operation, e.g. `string.ascii_letters * 2`.

Or be less explicit about the exact refcounts in the corresponding documentation than saying "will display". :)

Reply:
I'd like to keep the wording as is. I've changed the example to use a dict, in case CPython 3.9+ gets smarter about optimizing constant str initializations.